src/main/java/com/okta/sdk/framework/ApiClient.java

```diff
@@ -64,9 +61,20 @@
     ////////////////////////////////////////////
 
     public ApiClient(ApiClientConfiguration config) {
+        Proxy proxy = ProxySelector.getDefault().select(URI.create(config.getBaseUrl())).iterator().next();
+        URI proxyUri = URI.create(proxy.type() + "://" + proxy.address());
+        HttpRoutePlanner routePlanner;
+        if (Proxy.Type.HTTP.equals(proxy.type())) {
```

Comment:
This is intentional. Commons HttpClient only supports HTTP proxies so if there is a non-HTTP proxy (say SOCKS) it will fallback on connecting directly.